source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
-
-Running a Simple Job Script
----------------------------
-
-This script assumes that the data and executable are staged to
-/gpfs/f5/<project>/scratch/$USER. The scripts and data are located at
-/usw/user_scripts/.
-
-1. Use gcp to get the skeleton script from /usw/user_scripts/runscript
-   to your local home directory.
-
-.. code-block:: shell
-
-  gcp /usw/user_scripts/runscript ~$USER/
-
-2. Use gcp to get other files from /usw/user_scripts/ to your gpfs directory.
-
-.. code-block:: shell
-
-  gcp -r /usw/user_scripts/ /gpfs/f5/<project>/scratch/$USER/runscript
-
-3. Open the runscript.
-
-.. code-block:: shell
-
-  vim ~$/gpfs/f5/<project>/scratch/$USER/runscript
-
-The comments in the script will help you understand what each item does.
-
-4. Return to the directory where you copied the run script, and submit
-   your job.
-
-.. code-block:: shell
-
-  sbatch /gpfs/f5/<project>/scratch/$USER/runscript
-
-Make sure that the sbatch directives (--account, --walltime) have been
-changed.
-
-**Once the job is submitted**, you can use the following commands to
-check on your job.
-
-- To view job status:
-
-.. code-block:: shell
-
-  squeue -u $USER
-
-- For a detailed status check, use the scontrol commnand, and replace
-  "jobid" with your job's id.
-
-.. code-block:: shell
-
-  scontrol show job <jobid>
-
-For example:
-
-.. code-block:: shell
-
-  scontrol show job 123456789
-
-Once the job is finished, it should produce an output file.
-
-System Architechture
-====================
-
-Gaea is the largest of the NOAA research and development HPC systems,
-and is operated by DOE/ORNL.
-
-
-.. figure:: /images/GaeaC5.png
-
-The aggregate Gaea system:
-
-- consists of 245,760 AMD cores;
-- contains 646 TiB of memory
-- can perform 13.7 petaflops, or 13,700 trillion calculation each
-  second.
-
-Node Types
-----------
-
-- **Compute Nodes (C5):** 128 cores, HPE EX Rome, 251GB memory, run
-  model executable, filesystem mount - F5
-- **Batch Nodes:** 2 cores, 8GB memory, run scripts only (cores are
-  not charged)
-
-.. Note::
-
-  Batch Nodes are not very powerful. Do not write code/jobs that will use Batch nodes to do CPU intensive work
-
-- **ESLogin Nodes:**  32 cores, 512GB memory, run interactive
-  sessions, Matlab, compiles
-- **LDTN Nodes:** 16 cores, 24GB memory, I/O intensive jobs (combines,
-  etc.)
-- **RDTN Nodes:** 8 cores, 48GB memory, Data transfer jobs
-
-Clusters
---------
-- **C5** Gaea compute partition. Please see "System Architecture" and
-  "Hardware" for details.
-- **es** login nodes, local data transfer node queue (ldtn) and remote
-  data transfer node queue (rdtn)
-
-
-Examples:
-
-.. code-block:: shell
-
-  sbatch --clusters=c5 scriptname
-  #SBATCH --clusters=c5
-
-.. code-block:: shell
-
-  sbatch --clusters=es scriptname
-  #SBATCH --clusters=es
-
+    Use the ``-craype-verbose`` flag to display the full include and link
+    information used by the Cray compiler wrappers. This must be called on a
```

Comment:
Recommend:
If you want to see the full output you must call this on a file. For example...

Reply:
Adjusted in a new pushl.